docs/openapi.yaml

```diff
@@ -459,7 +459,47 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
-
+  /report/{institutionIdentifier}:
```

Comment:
Is the type of path parameter `institutionIdentifier` a URI? If yes, make it clearer? (You are testing with a URI in `FetchReportHandlerTest`)

Reply:
The name `institutionIdentifier` can make it seem like it is some other type of identifier